.gitignore

```diff
@@ -1 +1,5 @@
 node_modules
+yarn-error.log
+package-lock.json
+*.js
```

Comment:
`dist/*.js` and `dist/*.js.map` should be committed (we're not publishing the package to npm yet)